api.swagger.yaml

```diff
@@ -16753,6 +16774,12 @@ components:
           example: '01234567-89ab-cdef-1234-56789abcdef0'
           type: string
           format: uuid
+        attachments:
```

Comment:
This should be an array of uuid objects: [ { uuid: 'wefewfwef' }, { uuid: 'erghergerg'} ]